### PR TITLE
swarm: release v0.4.2

### PR DESCRIPTION
**Pull request by nonsense**

### CODE REVIEW

**Comment by nonsense** [comment]
@skylenet @acud not sure how we want to handle the CHANGELOG.md , but I see a few options:
- I don't want the updates to be directly part of the PRs, because then we'd be conflicting a lot on this file.
- developers could just push a commit to this branch after their PRs are merged, and prior to a release we merge this?
- whoever is responsible for a given release, scans Github PRs based on the milestore and copies most/all PRs here before making a release...

Something like that. Ideas?

**Comment by skylenet** [comment]
For me it's **mandatory** that the release manager reviews and updates the changelog.

For the rest: I kinda thing **it's nice** if someone else updates the release notes (either with a separate PR or within the same PR of the bug/feature), specially **if they introduced a change that requires some additional notes there**.

**Review by skylenet — APPROVED** [review]

**Review by acud — APPROVED** [review]

**Pull request merged**
